packages/logging_cloudwatch/aws_logging_cloudwatch/lib/src/cloudwatch_logger_plugin.dart

```diff
@@ -32,7 +30,7 @@ import 'package:meta/meta.dart';
 // The maximum number of log events in a batch is 10,000.
 
 const int _maxNumberOfLogEventsInBatch = 10000;
-const int _maxLogEventsTimeSpanInBatch = 24 * 3600;
+const int _maxLogEventsTimeSpanInBatch = 24 * 3600 * 1000;
```

Comment:
Prefer typed values to avoid errors like this

```suggestion
const Duration _maxLogEventsTimeSpanInBatch = Duration(hours: 24);
```